### DISCUSSION TITLE
Add Power Limit Information P(V)

### DISCUSSION THREAD

**Discussion by Kap3** [WillCodeForCats/solaredge-modbus-multi#587] [Ideas]
Hi,
I activate P(V) functions to reduce power when my AC voltage is too high.
Can you add this parameter for Home assistant?
!
[2024-04-19_150629](https://github.com/WillCodeForCats/solaredge-modbus-multi/assets/32192829/895a8a5d-bdb0-4056-b4ff-240730111c0c)

THanks

**Comment by WillCodeForCats** [comment]
Yes the integration already has this if the inverter supports it over modbus.

**Reply by Kap3** [reply]
I have SE6000H-RW000BNN4 with 0004.0020.0036.
With modbus connection but I can't find this setting. My inverter isn't supported?
Thanks

**Reply by WillCodeForCats** [reply]
You can turn on debug to confirm, but no, if you don't see it then your inverter does not support controls over modbus. Controls over modbus are not supported by all inverters.

**Reply by Kap3** [reply]
I have a controls over modbus but I dont'have  Power limit information in real time.
[config_entry-solaredge_modbus_multi-c18ecc9210c874cf651499eb5fe03693.json](https://github.com/WillCodeForCats/solaredge-modbus-multi/files/15375907/config_entry-solaredge_modbus_multi-c18ecc9210c874cf651499eb5fe03693.json)

![immagine](https://github.com/WillCodeForCats/solaredge-modbus-multi/assets/32192829/c2d3eff8-09fa-4906-8e77-dd5ad185adb7)

**Reply by WillCodeForCats** [reply]
Yes it's right there on your screen, active power limit.

**Reply by Kap3** [reply]
It's not work fine:
![immagine](https://github.com/WillCodeForCats/solaredge-modbus-multi/assets/32192829/72971e07-2acb-4dea-a4c1-2f1a9f182350)

It work only if I move Active Power slider. But If the inverter reduce the power for high temperature(>50°C) or for high voltage( I actived P(V) function) in home assistant I read every time 100%.

**Reply by WillCodeForCats** [reply]
Try changing Reactive Power Mode to RRCR and then press Commit Power Settings.

**Reply by cyberkryten** [reply]
I've found that value is updated by the inverter itself, so whilst you can change it, it gets updated immediately by the inverter, so it isn't something you can set to have any effect.

Currently trying to get some sense out of SE on how that works, why it changes etc as it is causing me issues because my Inverter is changing it to zero!  I thought I could change it back, but it immediately reverts to the inverter set value (RCRR is on).

It seems like it should actually be a ReadOnly parameter (at least the way it behaved on my inverter).

Here's my last 24 hours - this is without me making any changes to the value, that's all inverter made changes.

![Active Power Limit readings](https://github.com/WillCodeForCats/solaredge-modbus-multi/assets/99178799/982c370e-8888-4480-92eb-b0a0fb06c931)

**Reply by Kap3** [reply]
Now I actived RRCR mode and I'll study what it is! 
I 'll check tomorrow with sun! 
Thanks.

**Reply by Kap3** [reply]
I can't change it:
![immagine](https://github.com/WillCodeForCats/solaredge-modbus-multi/assets/32192829/75b4caf7-d153-4dee-8170-43eb4e200620)
after some minutes it return to default setting. 
Can I try to edit with set app ( I have installer account too)?
